packages/crypto/lib/models/block.js

```diff
@@ -162,7 +162,7 @@ module.exports = class Block {
 
   static getId (data) {
     const idHex = Block.getIdHex(data)
-    return new Bignum(idHex, 16).toString()
+    return new Bignum(idHex, 16).toFixed()
```

Comment:
bignumify or with parenthesis like this `(new Bignum(idHex, 16)).toFixed()`, not sure it gives the same result without parenthesis :thinking:

Reply:
This is the crypto package, `bignumify` is inside the `core-utils` which has a dependency of `crypto` so it would create a circular dependency.

Reply:
true :ok_hand: